x-pack/plugins/index_management/public/application/components/component_templates/component_template_selector/components/filter_list_button.tsx

```diff
@@ -67,26 +67,28 @@ export function FilterListButton({ onChange, filters }: Props) {
   );
 
   return (
-    <EuiPopover
-      ownFocus
-      button={button}
-      isOpen={isPopoverOpen}
-      closePopover={closePopover}
-      panelPaddingSize="none"
-      data-test-subj="filterList"
-    >
-      <div className="euiFilterSelect__items">
-        {Object.entries(filters).map(([filter, item], index) => (
-          <EuiFilterSelectItem
-            checked={(item as Filter).checked}
-            key={index}
-            onClick={() => toggleFilter(filter)}
-            data-test-subj="filterItem"
-          >
-            {(item as Filter).name}
-          </EuiFilterSelectItem>
-        ))}
-      </div>
-    </EuiPopover>
+    <EuiFilterGroup className="componentTemplates__filterListButton">
+      <EuiPopover
+        ownFocus
+        button={button}
+        isOpen={isPopoverOpen}
+        closePopover={closePopover}
+        panelPaddingSize="none"
+        data-test-subj="filterList"
+      >
+        <div className="euiFilterSelect__items">
```

Comment:
We normally don't use EUI classes directly here. But I noticed that this was already there and with the time we have left until 7.14 it's ok.

Reply:
I am not sure why this class was added. I can have a look into it. Will try to merge this first though.